dotnet-tests/UniffiCS.BindingTests/TestTraits.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using uniffi.traits;
+
+namespace UniffiCS.BindingTests;
+
+public class TestTraits
+{
+    [Fact]
+    public void TraitsWoring()
+    {
+        foreach (var button in TraitsMethods.GetButtons())
+        {
+            var name = button.Name();
+            Assert.Contains(name, new string[] {"go", "stop"});
+            Assert.Equal(TraitsMethods.Press(button).Name(), name);
+        }
+    }
+
+}
```

Comment:
Would be nice to also test that the native implementation of Button works as well, just like in the upstream version of the test